onnxruntime/contrib_ops/webgpu/quantization/matmul_nbits.cc

```diff
@@ -22,6 +22,9 @@ namespace {
 
 constexpr unsigned int kMinMForTileOptimization = 4;
 
+#define CEIL_DIV(numerator, denominator) \
+  (((numerator) + (denominator) - 1) / (denominator))
+
```

Comment:
could you change this to an inline function? it will ensure if denominator is function call it will be called only once.